server/routes/notification.js

```diff
@@ -114,9 +116,10 @@ router.get(
       Promise.all([
         // This should be replaced with a non-paged call that shows only recent notifications.
         // This isn't an audit!!
-        notificationService.getShiftNotifications(req.user.username),
-        notificationService.getShiftNotificationsPaged(req.user.username, offset, perPage),
-      ]).then(([count, rows]) => {
+        DEPRECATEnotificationService.getShiftNotifications(req.user.username),
+        DEPRECATEnotificationService.getShiftNotificationsPaged(req.user.username, offset, perPage),
+        notificationService.getPreferences(req.user.token),
```

Comment:
opportunity here to de-struct the vars we're using out of the `req` object. I appreciate this is more lift and shift so very much optional.